content/acra/configuring-maintaining/general-configuration/acra-server/_index.md

```diff
@@ -249,6 +249,17 @@ weight: 3
 
   Enable HTTP API.
 
+* `--http_api_use_tls={true|false}`
+
+  Enables the TLS for the HTTP API. Use together with the `--http_api_enable=true`.  Uses the same key and certificates as the Acra proxy.
+
+  Default is `false`.
+
+  {{< hint warning >}}
+  **Note:**
+  The default configuration with `--http_api_use_tls=false` is **insecure**. Enable TLS whenever possible.
```

Comment:
then why default is false? :)

Reply:
Backward compatibility ☹️
I guess, in the future we will make it `true` by default. However it's not that straightforward, because the clients should specify TLS keys and certificates, which they don't always do (like in situations when they play with the database and acra locally, so there is no need of specifying the TLS params).

Reply:
okay.jpg

Reply:
hm. looked one more time and see http_api_use_tls and http_api_enable... and I think that we should rename it to something like http_api_tls_transport_enable... I suggested it @G1gg1L3s in Slack and will repeat it in #550. So lets merge this PR after merge into acra repo.